atom/app/atom_content_client.cc

```diff
@@ -8,9 +8,46 @@
 #include <vector>
 
 #include "atom/common/chrome_version.h"
+#include "atom/common/options_switches.h"
+#include "base/command_line.h"
+#include "base/strings/string_split.h"
+#include "base/strings/string_util.h"
```

Comment:
yeah it was residual from previous usage to manually parse `flash_version` or provide one if not.